rpc/debug/trace.go

```diff
@@ -0,0 +1,49 @@
+// +build evm
+
+package debug
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/eth"
+	"github.com/ethereum/go-ethereum/eth/ethapi"
+	"github.com/ethereum/go-ethereum/eth/tracers"
+	"github.com/pkg/errors"
+
+	"github.com/loomnetwork/loomchain/store"
+	"github.com/loomnetwork/loomchain/txhandler/middleware"
+)
+
+func TraceTransaction(
+	app middleware.InMemoryApp,
+	blockstore store.BlockStore,
+	blockNumber, txIndex int64,
+	config eth.TraceConfig,
+) (interface{}, error) {
+	if err := app.RunUpTo(blockNumber, txIndex); err != nil {
```

Comment:
Here you should replay the txs using the `abci.Application` interface, so you'd call `app.BeginBlock`, then `app.DeliverTx` for each tx that needs to be replayed.